6.analyze_features/scripts/9a.mean_aggregated_direct_hypothesis_test.py

```diff
@@ -217,21 +248,44 @@
 
     # get counts of each feature group and the hypothesis test bool
     hypothesis_tests_df = (
-        df.groupby(["feature_group", "hypothesis_test_bool"])
+        df.groupby(
+            [
+                "feature_group",
+                "variance_pattern_bool",
+                "significance_bool_holm_bonferroni",
+                "hypothesis_test_bool",
+            ]
+        )
         .size()
         .reset_index(name="counts")
     )
     hypothesis_tests_df["permutation"] = data_shuffle_df
     list_of_enrichment_dfs.append(hypothesis_tests_df)
 final_enrichment_df = pd.concat(list_of_enrichment_dfs)
 final_enrichment_df.reset_index(drop=True, inplace=True)
-final_enrichment_df.head(15)
 
 
-# In[12]:
+# In[13]:
+
+
+final_enrichment_half_baked_df.head(60)
+
+
+# In[14]:
+
+
+final_enrichment_df.head(60)
```

Comment:
Why 60 for the head? Might consider reducing this if not necessary to make the notebook more simple.